automated/linux/kselftest/kselftest.yaml

```diff
@@ -50,6 +50,8 @@ params:
     BOARD: ""
     BRANCH: ""
     ENVIRONMENT: ""
+    # Set arbitrary environment variable
```

Comment:
Is it "arbitrary environment variables" or is there an example you can give here too for clarity?

Reply:
for the yaml it is single variable the way parsing is done and how the argument passing is hardcoded to `-E ${ENV}`, for the script user can pass `-E <env1> -E <env2>` multiple times for each env var

Reply:
Would it be possible to add an example into the # comment?